### ISSUE TITLE
Replace 'close' button with close icon

### ISSUE THREAD

**Issue by johngruen** [internetarchive/FirefoxNoMore404s#15]
Per the UX Spec:
![wayback version one 2x](https://cloud.githubusercontent.com/assets/3323249/16258984/a4e351dc-3860-11e6-8daf-8d69944d75ec.png)
Here's an svg of the close icon: https://www.dropbox.com/s/0d539agcasms4s3/Close.svg?dl=0

Button title attr should be set to 'close' for improved accessibility

**Comment by johngruen** [comment]
Good to close this.
